core/src/main/java/io/confluent/rest/RestConfig.java

```diff
@@ -93,13 +107,80 @@
       "<code>MetricReporter</code> interface allows plugging in classes that will be notified " +
       "of new metric creation. The JmxReporter is always included to register JMX statistics.";
   protected static final String METRICS_REPORTER_CLASSES_DEFAULT = "";
+  public static final String SSL_KEYSTORE_LOCATION_CONFIG = "ssl.keystore.location";
```

Comment:
@ijuma, do you know of any issues we may encounter if we made special helper methods for creating security configurations (with a prefix, eg in the schema registry, `kafkastore.ssl.*`)? Thinking about this more, I think this will be straightforward. Let me know if you disagree. Thanks!